docs/getting-started-with-react/getting-started-with-react.stories.mdx

```diff
@@ -89,7 +89,7 @@ For this example we are going to build a card that allows you to enter a topic
 and then search for a related gif.
 
 <Canvas>
-  <Story name="Composing Components">
+  <Story name="Gif Gift">
```

Comment:
I played around with this for ~20mins to try to prevent the split of a Docs page and a page with this story but still having it render nicely in this docs page and couldn't get it. At least now it has a better name.

Reply:
haha I don't get why this is its own section. that can't be by design. something must have changed in a storybook upgrade or something.